src/ui/find-ui/src/FamilyHubs.ServiceDirectory.Web/Errors/PossibleErrors.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections.Immutable;
+using FamilyHubs.SharedKernel.Razor.ErrorNext;
+using FamilyHubs.SharedKernel.Services.Postcode.Model;
+
+namespace FamilyHubs.ServiceDirectory.Web.Errors;
+
+public static class PossibleErrors
+{
+    public static readonly ImmutableDictionary<int, PossibleError> All = ImmutableDictionary
+        .Create<int, PossibleError>()
+        .Add(PostcodeError.NoPostcode, "You need to enter a postcode")
+        .Add(PostcodeError.InvalidPostcode, "Your postcode is not recognised - try another one")
```

Comment:
Are these messages from the prototype or what has been asked to use?

I would say that:
- Your postcode is not recognised - try another one -> Your postcode is not in a recognised format
-  You need to enter a valid postcode -> The postcode provided has no matches (or something like that?)

Reply:
They're the existing messages (without the hints as those aren't GDS compliant)
If you want to reword them better going through design.